test/development/acceptance/hydration-error.test.ts

```diff
@@ -36,13 +36,23 @@ describe('Error overlay for hydration errors', () => {
 
     expect(await session.hasRedbox()).toBe(true)
 
-    expect(await session.getRedboxDescription()).toMatchInlineSnapshot(`
+    if (process.env.TURBOPACK) {
+      expect(await session.getRedboxDescription()).toMatchInlineSnapshot(`
       "Error: Text content does not match server-rendered HTML.
 
       Warning: Text content did not match. Server: "server" Client: "client"
 
       See more info here: https://nextjs.org/docs/messages/react-hydration-error"
     `)
+    } else {
+      expect(await session.getRedboxDescription()).toMatchInlineSnapshot(`
+      "Error: Text content does not match server-rendered HTML.
```

Comment:
this snapshot looks indentical to https://github.com/vercel/next.js/pull/61077/files#diff-fdb3f499059998eeac833bd3858f5a920a2362cec03ad8d410672a4fc12daf37R41?